index.html

```diff
@@ -35,6 +36,99 @@ <h2>
             <li><img class="M" src="Images/M.png" alt="M">
         </ul>
     </main>
-    
+    </div>
+    <div class="Works">
```

Comment:
In our requirements for the project, the works section should be wrapped in a `section` tag rather than a `div`. Kindly fix it to meet the expected requirements 😁